portfolio/src/main/webapp/style.css

```diff
@@ -165,28 +164,28 @@ input[type="checkbox"]:checked ~ .sidebar {
   box-shadow: 0px 5px 5px rgba(0, 0, 0, 0.5);
 }
 
-.sidebar .social_media .fab {
+.social_media .fab {
     font-size: 18px;
     line-height: 30px;
 }
 
-.sidebar .social_media a:hover {
+.social_media a:hover {
   transform: rotate(0deg) skew(0deg) translate(0, -4px);
 }
 
-.sidebar .social_media li:nth-child(1) a:hover .fab {
+.social_media li:nth-child(1) a:hover .fab {
```

Comment:
Using nth-child here is not quite appropriate because it makes your CSS hard-coded to the structure of your HTML.

An appropriate use of nth-child is when you specifically want the 2nd child to be this color for some reason. But here it's actually based on which icon it is, not its position. It'd be better to use actual classes here, like a sort of instagram icon class and a facebook icon class, etc.